.github/workflows/upload.yml

```diff
@@ -30,4 +30,4 @@ jobs:
           cd repo
           git stash
           git pull --force origin master
-          make clean deploy
+          make deploy
```

Comment:
Not sure about this

Reply:
Any particular reason?

Reply:
Well even if `make clean deploy` adds more latency, I think it should still auto-clean everything. But I'm ok with it deciding what to clean as well.

Reply:
That's really how blogit is meant to be used tho. `clean` here means all previous recipe pages are destroyed, but the Makefile is relatively smart, it knows exactly which one to rebuild, which tag page to rebuild, etc. The only reason why I put `clean` in the first place was because I was still heavily modifying the Makefile, and when the Makefile is changed, you do need to rebuild everything (since the Makefile is not tracking itself as a dependency).

Reply:
Ok. Then I'm in agreeance with this change. Good explanation. Thanks.